### PR TITLE
parameters update

### PR DESCRIPTION
**Pull request by jlpoltrack**
Update for:

- Mode
- Tx + Rx Diversity
- Tx Ch Source

### CODE REVIEW

File: docs/PARAMETERS.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ Transmission power in Watts.
 
 #### Tx Diversity #### 
 Diversity mode. 
-Can be "enabled", "antenna1", "antenna2". 
+Can be "enabled", "antenna1", "antenna2", "r:en, t:ant1", "r:en, t:ant2".
 
 #### Tx Ch Source #### 
 Selects the source from which the rc data should be read. 
-Can be "none", "mbridge", "in", "crsf".
+Can be "none", "mbridge", "in", "crsf", "elrs rx".
```
Comment (olliw42) [inline docs/PARAMETERS.md:43]: this option is not going to come
it was for a special version for one potential user, but - AFAIK - has not even been tried

what has changed is the sequence of options, if I recall correctly mbridge and crsf were swapped :)

Comment (jlpoltrack) [reply docs/PARAMETERS.md]: Updated

Comment (olliw42) [reply docs/PARAMETERS.md]: should be  "none", "crsf", "in", "mbridge" I think, https://github.com/olliw42/mLRS/blob/main/mLRS/Common/setup_list.h#L95
:)

File: docs/PARAMETERS.md [hunk 2]
```diff
@@ -36,11 +36,11 @@ Transmission power in Watts.
 
 #### Tx Diversity #### 
 Diversity mode. 
-Can be "enabled", "antenna1", "antenna2". 
+Can be "enabled", "antenna1", "antenna2", "r:en, t:ant1", "r:en, t:ant2".
```
Comment (olliw42) [inline docs/PARAMETERS.md:39]: I guess could be useful if there would be some sort of brief explanation. If not here, maybe somewhere else?
No blocker for sure.

Comment (jlpoltrack) [reply docs/PARAMETERS.md:39]: Good idea - added a line for both Tx and Rx parameters explaining what that does.

**Comment by olliw42** [comment]
LGTM :)

**Pull request merged**